sonaranalyzer-dotnet/rspec/cs/S4564_c#.html

```diff
@@ -0,0 +1,36 @@
+<p>ASP.NET is coming with a feature looking at HTTP requests to prevent potentially dangerous content to perform a cross-site scripting (XSS) attack.
+There is no reason to disable this mechanism even if it should not be the only one in place to prevent XSS attacks.</p>
```

Comment:
Suggested wording change: _There is no reason to disable this mechanism even if other checks to prevent XXS attacks are in place_.

Reply:
@agigleux Could you update the RSPEC?

Reply:
Done.